pom.xml

```diff
@@ -4,7 +4,7 @@
 
 	<groupId>org.bahmni.module</groupId>
 	<artifactId>bahmni.ie.apps</artifactId>
-	<version>1.1.0-SNAPSHOT</version>
+	<version>1.2.0-SNAPSHOT</version>
```

Comment:
@angshu can you revert this to be back to 1.1.0-SNAPSHOT?

Reply:
resolved